src/rofl/common/crofbase.cc

```diff
@@ -622,8 +609,9 @@ void
 crofbase::handle_connect_refused(
 		crofctl& ctl, crofconn& conn)
 {
-	journal.log(LOG_INFO, "connection refused: ctlid=%s, auxid=%s, raddr=%s",
-			ctl.get_ctlid().str().c_str(), conn.get_auxid().str().c_str(), conn.get_raddr().str().c_str());
+	VLOG(2) << "connection refused: ctlid=" << ctl.get_ctlid().str().c_str()
+	   	<< ", auxid=" << conn.get_auxid().str().c_str() << ", raddr=" << 
```

Comment:
just kept them as they were, but we should overall align the logging output separately from this